src/ui/LogFormatWidget.cpp

```diff
@@ -34,6 +34,7 @@
 #include <QMessageBox>
 #include <QPushButton>
 #include <QToolTip>
+#include <QtWidgets>
```

Comment:
This includes *all* the header files of the QtWidget library, including only QAction should be enough.

Reply:
Fixed